src/fs/mod.rs

```diff
@@ -9,3 +9,10 @@ pub mod feature;
 pub mod fields;
 pub mod filter;
 pub mod mounts;
+
+use lazy_static::lazy_static;
+use std::collections::HashMap;
+use std::sync::Mutex;
+lazy_static! {
+    static ref RECURSIVE_SIZE_HASHMAP: Mutex<HashMap<u64, u64>> = Mutex::new(HashMap::new());
```

Comment:
Not required but it would be interesting to compare this to a DashMap.

Reply:
DashMap performance was slower from my tests compared to the HashMap, maybe due to the tuple key introduced.